azure-pipelines.release-fluentui.yml

```diff
@@ -16,23 +22,31 @@ steps:
       versionSpec: '10.x'
     displayName: 'Install Node.js'
 
+  - script: |
+      ver=`node -p "require('./packages/fluentui/react-northstar/package.json').version"`
+      echo "##vso[task.setvariable variable=deployBasePath]$ver"
+      echo "##vso[task.setvariable variable=docsiteVersion]$ver"
+      echo "Docsite base path published to for version $ver"
+    displayName: Set base path and version variables for job
   - script: |
       git config user.name "Fluent UI Build"
       git config user.email "fluentui-internal@service.microsoft.com"
       git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/microsoft/fluentui.git
     displayName: Authenticate git for pushes
-
   - script: |
-      npx midgard-yarn install
```

Comment:
Why this is changed?

Reply:
It wasn't working in the build for 47 when I first made these changes, I will revert and see if that works